src/main/metanorma/sections/informative/geometry/index.adoc

```diff
@@ -4,10 +4,10 @@
 The GeoAPI geometry packages use the `opengis​.geometry` namespace
 and are placeholder for types defined in the <<ISO19107>> - _Spatial schema_ specification.
 #TODO: upgrade to ISO 19107:2019.#
```

Comment:
@desruisseaux not sure if you knew about these TODOs, but I can't really help with them. Flagging for your information.

Reply:
Hello Peter. Yes I'm aware of them. I think they will be work for another version after 3.1. Thanks for your review!